contrib/local-environment/oauth2-proxy-keycloak.cfg

```diff
@@ -0,0 +1,20 @@
+http_address="0.0.0.0:4180"
+cookie_secret="OQINaROshtE9TcZkNAm-5Zs2Pv3xaWytBmc5W7sPX7w="
+email_domains=["example.com"]
+cookie_secure="false"
+upstreams="http://httpbin"
+cookie_domains=[".localtest.me"] # Required so cookie can be read on all subdomains.
+whitelist_domains=[".localtest.me"] # Required to allow redirection back to original requested target.
+
+# keycloak provider
+client_secret="72341b6d-7065-4518-a0e4-50ee15025608"
+client_id="oauth2-proxy"
+redirect_url="http://oauth2-proxy.localtest.me:4180/oauth2/callback"
+
+# in this case oauth2-proxy is going to visit
+# http://keycloak.localtest.me/auth/realms/master/.well-known/openid-configuration for configuration
+oidc_issuer_url="http://keycloak.localtest.me:8080/auth/realms/master"
+provider="oidc"
```

Comment:
It's interesting that you've chosen to use the OIDC provider for this rather than the keycloak provider, have you tried the keycloak provider? Are you aware of any differences?

Reply:
I tried it first following https://oauth2-proxy.github.io/oauth2-proxy/auth-configuration#keycloak-auth-provider, but don't remember why I didn't work. Also with `provider="keycloak"` you have to  configure more URLs. 

I'm not sure if with `provider=oidc` you can still use `keycloak-group` option. I guess not.

Reply:
Yes I also suspect not, I wonder if we can end up removing the keycloak provider if this works, I guess we will have to work out the group info first